src/modules/Addcomment.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable  */
```

Comment:
- [ ] Kindly do not disable linters as they help us write better and cleaner code